config/application.yml.default

```diff
@@ -199,6 +199,7 @@ max_mail_events_window_in_days: 30
 max_phone_numbers_per_account: 5
 max_piv_cac_per_account: 2
 min_password_score: 3
+monthly_key_metrics_report_configs: '[]'
```

Comment:
is this used at all?

Reply:
No, this has been replaced by `team_agnes_email` and `team_all_feds_email`.  I’m removing it. 👍